lib/jekyll-admin/server.rb

```diff
@@ -75,7 +75,11 @@ def write_file(path, content)
       File.open(path, "wb") do |file|
         file.write(content)
       end
-      site.process
+      if ENV["RACK_ENV"]
+        site.process
+      else
+        site.read
+      end
```

Comment:
@parkr any bright ideas? We'd need to force `watch` to false. Perhaps the same way we do for the Pages gem?